src/artemis/devices/I03Smargon.py

```diff
@@ -0,0 +1,11 @@
+
+from ophyd import Device, EpicsMotor
+from ophyd import Component as Cpt
+
+class I03Smargon(Device):
```

Comment:
**should**: We actually already have the smargon in the codebase, it was just badly named. I've renamed it at https://github.com/DiamondLightSource/python-artemis/pull/112/commits/488143aacf30099d4f48c8354ea6767a284d1aa7